models/user.go

```diff
@@ -1778,25 +1792,23 @@ func synchronizeLdapSSHPublicKeys(usr *User, s *LoginSource, sshPublicKeys []str
 }
 
 // SyncExternalUsers is used to synchronize users with external authorization source
-func SyncExternalUsers(ctx context.Context) {
+func SyncExternalUsers(ctx context.Context, updateExisting bool) error {
 	log.Trace("Doing: SyncExternalUsers")
 
 	ls, err := LoginSources()
 	if err != nil {
-		log.Error("SyncExternalUsers: %v", err)
-		return
+		log.Trace("SyncExternalUsers: %v", err)
```

Comment:
It's a Trace because the caller above is going to place the same Error in the Notices table and it just felt like over logging to have it log at Error - Can change it back though if preferred.

Reply:
The log can be scanned by tools.... it may be useful to have the error in there too.

Reply:
```suggestion
		log.Error("SyncExternalUsers: %v", err)
```